app/views/layouts/_analytics_meta_tags.html.erb

```diff
@@ -0,0 +1,8 @@
+<%= tag.meta name: "govuk:components_gem_version", content: GovukPublishingComponents::VERSION %>
+
+<% if user_signed_in? %>
```

Comment:
Non-blocking comment: would it be better to add `&& current_user` to this conditional rather than using the safe navigation operator on each tag below?